util/install/install.Ubuntu.sh

```diff
@@ -1,53 +1,40 @@
 #!/usr/bin/env bash
-# DeepDive installers for Ubuntu Linux
+# DeepDive installers for Debian/Ubuntu Linux
 
-LSB=$(lsb_release -r 2>/dev/null)
-case ${LSB##*	} in
-    12.04|14.04|15.04) true ;;
-    *) error "Ubuntu $LSB found: This installer only works with Ubuntu 12.04, 14.04, and 15.04."
+LSB=$(lsb_release -ir | cut -f2)
```

Comment:
I think you need to add `-ri` flags to get normal output.  `lsb_release` seems to be always available on Debian-based systems.  Most base packages are depending on it, so should be installed.  Btw when `lsb_release` goes wrong, the current code will just show warnings and still let the user run the defined installers.  Maybe I should make it more explicit by adding a ` || true`.